### DISCUSSION TITLE
Unminimized/Compressed Next.js Builds on Vercel

### DISCUSSION THREAD

**Discussion by waxidiotic** [vercel/community#833] [Help]
I'm trying to debug some issues we're having on staging and production that are not able to reproduced locally. I'm wondering if there is a way to deploy a preview build on Vercel that is non-optimized, so it's essentially the same build that gets served up locally when developing?

I tried `optimization.minimize: false` in `next.config.js`.

**Comment by swarnava** [comment]
It sounds like you are probably looking for `vercel dev` command : https://vercel.com/docs/cli#commands/dev

**Reply by waxidiotic** [reply]
I don't believe so because we aren't using any serverless functions that would affect this so there's probably not much of a difference for us running `vercel dev` compared to `next dev`.

We figured out the issue was the use of A/B testing flags with LaunchDarkly. We could try to replicate that locally but it would also be nice to be able to deploy an unminified build.

**Reply by swarnava** [reply]
Have you tried  `config.optimization.minimize = false` in  in next.config.js to custom webpack config ? https://webpack.js.org/configuration/optimization/#optimizationminimize
